contributing/contributing-code.md

```diff
@@ -0,0 +1,115 @@
+# Contributing code and features
+
+Thank you for helping make AMP better by fixing bugs, adding features or improving the AMP code in some other way!
+
+This document describes the process you will go through to make a change in AMP.
+
+## Process Overview
+
+### Process for minor bug fixes and updates
+
+We want to make it as easy as possible to get in small fixes.  A fix for a small bug should be as easy as creating a PR with the change, adding/fixing a test, and sending it to a reviewer.
+
+- [ ] (optional) If you are fixing a bug and there is an existing GitHub issue, assign it to yourself or comment on it to let others know you are working on it.  If there is no GitHub issue consider filing one, but for minor fixes your Pull Request description may be enough.
+- [ ] (optional) [Find a reviewer](#find-a-reviewer) before you start coding to help you answer questions.  If you don't have any questions you can find a reviewer once you have a PR ready.
+- [ ] Follow the parts of the [Implementation](#implementation) section that makes sense for your change.  There are many parts of the process that you probably won't need to follow for a minor fix--e.g. you may not need to make validator changes or put your change behind an experiment for minor fixes.  If in doubt ask your reviewer.
+
+### Process for significant changes
+
+Significant changes (e.g. new components or significant changes to behavior) require consultation with and approval from knowledgeable members of the community.
+
+- [ ] *Before you start coding*, [find a reviewer](#find-a-reviewer) who you can discuss your feature with and who can help guide you through the process.
+- [ ] File an [Intent-to-implement (I2I)](TODO) GitHub issue and cc your reviewer on it.  The I2I should include:
```

Comment:
Fix todo?

Reply:
I've filed #20277 to add the I2I template, and put in a tentative link that I think will work here (I'll have to try it out once that template is merged in).

I need to do the same for the I2S template as well.